app/views/foreman_snapshot_management/hosts/_snapshots_tab.html.erb

```diff
@@ -1,3 +1,3 @@
-<% if @host.provider == 'VMware' %>
+<% if authorized_for(:controller => 'foreman_snapshot_management/snapshots', :action => :index) && @host.provider == 'VMware' %>
```

Comment:
I know, it is a little late, but i'd like to have a more generic test if the host_provider is able to take snapshots.
Maybe something like `Snapshot.applicable_for(@host)` implemented in the model.
It would help a lot if we implemented more providers in the future.
What do you think. Of course, we can do this in a separate pull-request if you prefer.

Reply:
Yeah, I already had the same idea but wanted to keep the changes small. We should also add more safety checks in the controller. I'll write something up.